multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -52,6 +52,7 @@
 MIN = "min"
 ACCURACY = "accuracy"
 ACC = "acc"
+NER_METRIC = "overall_accuracy"
```

Comment:
Is it possible that ner uses other metrics besides "overall_accuracy"? Is it better to define 
`OVERALL_ACCURACY = "overall_accuracy"`?

Reply:
yes, there are overall_f1, overall_precision, overall_recall, and for each entity type, we will also output the specific evaluation results. Users can directly use the entity group name to access the results. If no specific evaluation metric is given, the default output would be like:
```
 {'LOC': {'precision': 0.9313022700119474, 'recall': 0.9307462686567164, 'f1': 0.9310241863242759, 'number': 1675}, 
     'MISC': {'precision': 0.7767379679144385, 'recall': 0.8114525139664804, 'f1': 0.7937158469945355, 'number': 716}, 
     'ORG': {'precision': 0.8750719631548647, 'recall': 0.9074626865671642, 'f1': 0.8909730363423213, 'number': 1675}, 
     'PER': {'precision': 0.9567047504509922, 'recall': 0.958433734939759, 'f1': 0.957568462232922, 'number': 1660}, 
     'overall_precision': 0.9019237375472346, 'overall_recall': 0.9170450576318547, 'overall_f1': 0.9094215448562522, 'overall_accuracy': 0.9842387467323067}
```

Reply:
Since `NER_METRIC ` have several choices, e.g., overall_accuracy, overall_f1, overall_precision, or overall_recall, I would suggest no using it to refer to only `overall_accuracy`. Maybe it's better to define `OVERALL_ACCURACY = "overall_accuracy"` and replace `NER_METRIC` with `OVERALL_ACCURACY` in the code. In this way, we could easily use other metrics, e.g., `overall_f1`.

Reply:
Sounds good, I will add that.